<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RevelationRightSidebarClass</class>
 <widget class="QWidget" name="RevelationRightSidebarClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RevelationRightSidebar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0" colspan="3">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
           <item row="0" column="0">
            <widget class="FluLabel" name="label_2">
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="FluLabel" name="labelStatus">
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="FluLabel" name="label">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="FluLabel" name="label_3">
             <property name="text">
              <string>Tag:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="FluLabel" name="labelTag">
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="FluLabel" name="label_4">
             <property name="text">
              <string>Start Time:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="FluLabel" name="label_5">
             <property name="text">
              <string>Finish Time:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="FluLabel" name="label_6">
             <property name="text">
              <string>Deadline:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="FluCalendarDateTimePicker" name="btnSelectStartTime">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="FluCalendarDateTimePicker" name="btnSelectFinishTime">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="FluCalendarDateTimePicker" name="btnSelectDeadline">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="FluComboBox" name="cbType"/>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>83</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="FluLineEdit" name="editTitle"/>
         </item>
         <item row="1" column="0">
          <widget class="FluTextEdit" name="editDesc"/>
         </item>
         <item row="4" column="0">
          <widget class="FluPushButton" name="btnDelete">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="FluPushButton" name="btnAddToRoutine">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="FluPushButton" name="btnHide">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="FluLabel" name="labelCreateTime">
        <property name="text">
         <string>Created:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FluPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">FluPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>FluComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">FluComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>FluLabel</class>
   <extends>QLabel</extends>
   <header location="global">FluLabel.h</header>
  </customwidget>
  <customwidget>
   <class>FluTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">FluTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>FluLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">FluLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>FluCalendarDateTimePicker</class>
   <extends>QPushButton</extends>
   <header location="global">FluCalendarDateTimePicker.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
